### PR TITLE
[DinoMod] Dinosaur Exhibit Overhaul

### PR DESCRIPTION
**Pull request by LyleSY**
Cleanup monster groups, move field office out of town, rename

#### Summary

SUMMARY: Mods "Move DinoMod location to wilderness"

#### Purpose of change

Reduce dino/zombie collision, improve realism, reduce mod overhead, improve player experience

#### Describe the solution

Move field office to country as a special, completely redo zombie spawns to midgame and only dinos and move to monster groups and rename, remove random forest creature spawns, despecialize item groups to reduce overhead, remove sidewalk, revamp dinosaur groups to include more dino types, rename the whole thing to make it more clear to the player what it is and create two new location files to support it., keep overlay and obsolete fieldoffice to keep savegame compatibility

#### Describe alternatives you've considered

Chuck the thing entirely

#### Testing

Doesn't crash anymore! No load errors! Thanks to @ephemeralstoryteller for testing this. Spawns, largely works as intended, though Deinonychus do like to murder things. Loads into existing save with no errors

Old map with old location with new code:

Screen Shot 2020-06-02 at 9 52 59 PM

New map:

Screen Shot 2020-06-02 at 10 14 24 PM

#### Additional context

This is legacy content that hadn't been touched yet, working to integrate it into the rest of the mod and its story

### CODE REVIEW

**Comment by LyleSY** [comment]
Huge thanks to @curstwist wist for spelling out the specials weirdness to me, it's just not how my brain works

**Comment by LyleSY** [comment]
I'm sure I'm missing something obvious and stupid, but I can't find it. Can someone give it a look?

File: data/mods/DinoMod/overmap_specials.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "dinosaur exhibit",
+    "overmaps": [ { "point": [ 0, 0, 0 ], "overmap": "dinoexhibit" } ],
```
Comment (curstwist) [inline data/mods/DinoMod/overmap_specials.json:5]: adjust overmap value to: dinoexhibit_north since you allow it to rotate.

**Comment by LyleSY** [comment]
AppVeyor error doesn't appear to be related

**Comment by ZhilkinSerg** [comment]
Since `fieldoffice` overmap terrain is obsoleted and there is no migration defined it would cause issues when loading old savegames with fieldoffice on overmap.

It is not currently possible to do migration purely via json, but you can change name and mapgen of obsoleted terrain to that of field (instead of deleting overmap terrain) to not break savegame compatibility.

**Comment by LyleSY** [comment]
I think I've done that and I don't think the Travis errors are related to this PR

**Pull request merged**